src/structures/CryptoCurrency.ts

```diff
@@ -26,4 +27,20 @@ export default class CryptoCurrency {
     this.explorer = payload.explorer;
     this.format = new CryptoCurrencyFormat(payload.format);
   }
+
+  /**
+   * Convert a raw value to an amount.
+   * @param value The raw value
+   */
+  public convertToAmount(value: BigNumber): BigNumber {
```

Comment:
What about creating an alias for this function which is named `convertFromRaw`? @ZeroWave022

Reply:
We could just rename this one to `convertFromRaw`. Do we keep the function which does the opposite as `convertToRaw`?

Reply:
Yes